seedpod_ground_risk/main.py

```diff
@@ -74,7 +74,11 @@ def delete_layer(self):
         self._plot_worker.remove_layer(self._layer)
 
     def export_path_json(self):
-        self._plot_worker.export_path_json(self._layer, "paths")
+        from PySide2.QtWidgets import QFileDialog
+
+        file_dir = QFileDialog.getExistingDirectory(self, "Save plot image...", os.getcwd(), QFileDialog.ShowDirsOnly)
```

Comment:
Looks like you might have copied another dialog without changing the name. This one doesn't save a plot image!